circle.yml

```diff
@@ -1,18 +1,21 @@
+machine:
+  services:
+    - docker
+
 dependencies:
   override:
-    # Install tox and the pyenv plugin for tox so we can activate multiple
-    # Python binaries of the same major version onto the path at the same time
-    - pip install tox tox-pyenv
-    - pyenv local 2.7.6 3.4.3 3.5.0 pypy-2.5.0
-    - sudo apt-get install -y -qq libpython3.4
-
-    # Install the deadsnakes ppa repo so we can install libpython3.5
-    #- sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys DB82666C
-    #- sudo add-apt-repository -y ppa:fkrull/deadsnakes
-    #- sudo apt-get update
-
-    #- sudo apt-get install -y -qq libpython3.5
+    # We override the dependencies section so that CircleCI doesn't find the
+    # setup.py and install the dependencies. This is mostly an issue with
+    # libraries but can carry over into applications as well. (NZ)
+    - pwd
 
 test:
-  post:
-    - mv pytest-reports/ $CIRCLE_TEST_REPORTS/
+  override:
+    - ls -la
+    - /home/ubuntu/$CIRCLE_PROJECT_REPONAME/docker-run-tests
+
+deployment:
```

Comment:
Let's use the recommended bash script for this ([docs](https://docs.codecov.io/docs/about-the-codecov-bash-uploader)).

```yaml
deployment:
  codecov:
    branch: /.*/
    commands:
      - bash <(curl -s https://codecov.io/bash)
```